pkg/chartutil/values.go

```diff
@@ -288,6 +288,27 @@ func coalesceTables(dst, src map[string]interface{}) map[string]interface{} {
 	return dst
 }
 
+// ToRenderValues composes the struct from the data coming from the Releases, Charts and Values files
+func ToRenderValues(chrt *chart.Chart, chrtVals *chart.Config, options map[string]interface{}) (Values, error) {
```

Comment:
This is sorta frowned upon in Go (though we used to do it all the time in PHP). But @fibonacci1729 has a cool pattern for doing this in Go.